versioned/storage/cassandra/src/main/java/org/projectnessie/versioned/storage/cassandra/CassandraConstants.java

```diff
@@ -33,11 +33,11 @@ final class CassandraConstants {
 
   static final String COLS_COMMIT =
       "c_created, c_seq, c_message, c_headers, c_reference_index, c_reference_index_stripes, c_tail, c_secondary_parents, c_incremental_index, c_incomplete_index, c_commit_type";
-  static final String COLS_REF = "r_name, r_initial_pointer, r_created_at";
+  static final String COLS_REF = "r_name, r_initial_pointer, r_created_at, r_extended_info";
```

Comment:
Just curious: what's our global stance on DB schema evolution? Here, all pre-existing rows won't have any info in this new column. Also, `t_commit_id` column has been virtually deleted, but would still be physically present in the database. Is the suggestion to do a full export-upgrade-import?

Reply:
The backends tell users about the difference on startup, with the schema to apply.